### PR TITLE
system-modify: Add additional settings to be able to modify

### PR DESCRIPTION
**Pull request by jvillal-amp**
The system-create command has many more settings that can be set,
especially for power options.

Add these settings to the system-modify command so you can modify them
via the command line. Previously could only change them via the GUI.

Fixes: #36

### CODE REVIEW

File: Client/src/bkr/client/commands/cmd_system_modify.py [hunk 1]
```diff
@@ -140,6 +195,14 @@ def run(self, *args, **kwargs):
         if custom_policy:
             system_attr['active_access_policy'] = {'custom': True}
 
+        attrs = ['location', 'power_type', 'power_address', 'power_user',
+                 'power_password', 'power_id', 'power_quiescent_period',
+                 'release_action']
+        for attr in attrs:
+            value = kwargs.pop(attr, None)
+            if value is not None:
+                system_attrs[attr] = value
```
Comment (StykMartin) [inline Client/src/bkr/client/commands/cmd_system_modify.py:204]: `system_attrs` -> `system_attr`

Comment (jvillal-amp) [reply Client/src/bkr/client/commands/cmd_system_modify.py]: Done. And thanks!

**Comment by jvillal-amp** [comment]
I think this is ready to be run by CI:

```
# ./run-tests.sh -v bkr.inttest.client.test_system_modify
+ env BEAKER_CONFIG_FILE=server-test.cfg DISCARD_SUBPROCESS_OUTPUT=0 PYTHONPATH=../Common:../Server:../LabController/src:../Client/src:../IntegrationTests/src python2 -c '__requires__ = ["CherryPy < 3.0", "Jinja2 >= 2.6"]; import pkg_resources; from nose.core import main; main()' -v bkr.inttest.client.test_system_modify
test_cannot_change_active_policy_without_permission (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_condition (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_hypervisor (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_owner (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_power_address (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_power_id (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_power_password (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_power_quiescent (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_power_type (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_power_user (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_change_release_action (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_invalid_condition (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_invalid_system (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_modify_active_access_policy (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_modify_attributes_policy (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_no_changes (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok
test_no_fqdn (bkr.inttest.client.test_system_modify.ModifySystemTest) ... ok

----------------------------------------------------------------------
Ran 17 tests in 30.246s

OK
```

File: IntegrationTests/src/bkr/inttest/client/test_system_modify.py [hunk 2]
```diff
@@ -4,7 +4,8 @@
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 
-from bkr.server.model import session, SystemStatus, SystemPermission, Hypervisor
+from bkr.server.model import session, SystemStatus, SystemPermission, Hypervisor, \
```
Comment (StykMartin) [inline IntegrationTests/src/bkr/inttest/client/test_system_modify.py:7]: Do not use `\`. Instead of that, you can use `(` and `)`

**Comment by StykMartin** [comment]
One error in dogfood caused by #38

File: IntegrationTests/src/bkr/inttest/client/test_system_modify.py (hunk 2 above)
Comment (jvillal-amp) [reply IntegrationTests/src/bkr/inttest/client/test_system_modify.py]: Done

**Pull request merged**